docs/Course Wiki/ECE Course Offerings/ECE329.md

```diff
@@ -0,0 +1,55 @@
+# ECE329
+
+## Content Covered
+
+The first part of the course is a survey of electrostatics and Maxwell's equations, which are relevant to electromagnetic fields, in both differential and integral forms. They form the backbone of this course; knowing them from the beginning is advantageous as some later concepts are formulated within the same framework. Prior exposure to such topics is likely, either through PHYS212 or AP Physics, but ECE329 goes into far greater depth by leveraging the full power of Calculus 3 theorems to handle more general situations. Careful attention is a must, as it is easy to erroneously dismiss these critical topics as review. Students then learn about potential functions, electromagnetic wave behavior in material media such as conductors, dielectrics, and magnetic materials. The course emphasizes plane wave transmissions and their phasor forms; it is advisable to pay close attention to their mathematical derivations. Towards the end of the course, students learn about transmission lines and using Smith charts to determine reflection coefficients and input impedances.
```

Comment:
Can this be done more in a struct like bullet points and how many weeks is spent on each topic? I think this will be easier for people to read. We can also have this section but I think it would better if people can skim quickly